### DISCUSSION TITLE
How to hide sections from Table of contents

### DISCUSSION THREAD

**Discussion by J0B10** [squidfunk/mkdocs-material#4015] [Q&A] [answered]
Hey there! 

For a wiki I'd like to have the following Admonitions block with an example:

![grafik](https://user-images.githubusercontent.com/17405009/173233252-05e6fd36-486f-4f70-a59f-c48e97f0e2b3.png)

The example does contain a **heading**, that I don't want to show up in search or the Table of Contents (on the right side).

For hiding it from search I found [`{ data-search-exclude }`](https://squidfunk.github.io/mkdocs-material/setup/setting-up-site-search/#excluding-sections) metadata tag which is working fine. 

**But how can I hide it from the table of contents?** Is there any easy way to do this using metadata tags?

Only thing I found was using `{ data-search-exclude #markdown data-toc-label=''}` (see #2500) to rename the label to an empty string, but that leaves an empty line in the table.

### Used source code:
```md
!!! example
    ## Set Block: `setblock` { data-search-exclude }

    **persistent**, ==**static**== (1)
    { .annotate }

    Changes the block at the given position.

    1. This flag states that `setblock` can be used as static event.
```

**Comment by squidfunk** [comment, marked as answer]
Possible, see https://github.com/squidfunk/mkdocs-material/issues/313#issuecomment-299698639
